Include/cpython/abstract.h

```diff
@@ -39,6 +39,12 @@ PyAPI_FUNC(PyObject *) _Py_CheckFunctionResult(
     PyObject *result,
     const char *where);
 
+// Usage: assert(_Py_CheckSlotResult(obj, "__getitem__", result != NULL)));
+PyAPI_FUNC(int) _Py_CheckSlotResult(
```

Comment:
Why an API function?
You know that you will regret it later :wink: https://www.python.org/dev/peps/pep-0620/

Reply:
Oh, let me see that PEP... ah yes ;-) I wanted to define it close to _Py_CheckFunctionResult() definition, but you're right that it's a bad idea to export the symbol.